lib/grape/dsl/validations.rb

```diff
@@ -13,6 +13,9 @@ def reset_validations!
           unset_namespace_stackable :declared_params
           unset_namespace_stackable :validations
           unset_namespace_stackable :params
+          if route_setting(:description)
```

Comment:
I think your helper suggestion is good, I like it. If something is not used, nuke it.